requirements-test.txt

```diff
@@ -422,20 +422,11 @@ decorator==5.2.1 \
     #   -r requirements.txt
     #   dogpile-cache
     #   gssapi
-deprecated==1.2.18 \
-    --hash=sha256:422b6f6d859da6f2ef57857761bfb392480502a64c3028ca9bbe86085d72115d \
-    --hash=sha256:bd5011788200372a32418f888e326a09ff80d0214bd961147cfed01b5c018eec
-    # via
-    #   -r requirements.txt
-    #   opentelemetry-api
-    #   opentelemetry-exporter-otlp-proto-grpc
-    #   opentelemetry-exporter-otlp-proto-http
-    #   opentelemetry-semantic-conventions
-dogpile-cache==1.4.0 \
-    --hash=sha256:b00a9e2f409cf9bf48c2e7a3e3e68dac5fa75913acbf1a62f827c812d35f3d09 \
-    --hash=sha256:f1581953afefd5f55743178694bf3b3ffb2782bba3d9537566a09db6daa48a63
+dogpile-cache==1.4.1 \
+    --hash=sha256:99130ce990800c8d89c26a5a8d9923cbe1b78c8a9972c2aaa0abf3d2ef2984ad \
+    --hash=sha256:e25c60e677a5e28ff86124765fbf18c53257bcd7830749cd5ba350ace2a12989
     # via -r requirements.txt
-Flask==3.1.2 \
+flask==3.1.2 \
```

Comment:
can we get rid of dependency version upgrades from this PR?

Reply:
Yes, I'll do it